roles/openshift_health_checker/openshift_checks/etcd_volume_check.py

```diff
@@ -0,0 +1,68 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about etcd volume size.
+"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdImageData(OpenShiftCheck):
+    """Check that etcd volume capacity is not over 50%"""
+
+    name = "etcd_volume_check"
+    tags = ["etcd_volume_check"]
+
+    def run(self, _, task_vars):
```

Comment:
In this case I'd keep consistent with the "interface" we're implementing... in the absence of explicit types or other hints, I'd say this is more idiomatic:

```python
def run(self, tmp, task_vars):
```

This is the same signature as action plugins.

We should be passing `tmp` forward to a module that runs in the target.